xbmc/music/windows/GUIWindowMusicNav.cpp

```diff
@@ -287,6 +287,10 @@ bool CGUIWindowMusicNav::GetDirectory(const std::string &strDirectory, CFileItem
       OnRetrieveMusicInfo(items);
   }
 
+  //Navigating music files so default content to "songs" unless in sources folder.
+  //This content allows view type to include media info so that file tag data can be displayed
+  if (!URIUtils::IsSourcesPath(strDirectory))
```

Comment:
I get what you're trying to achieve but this way messes up the content type for root and sources which should both be 'empty' not 'songs'.